---
permalink: /research/
title: "Research Directions"
author_profile: true
redirect_from: 
  - /md/
  - /markdown.html
---

## Direction 1: Metabolic modeling of human gut microbiome
<img src="../images/Theme1_schematic.png" align="right" width="300px"/>
We aim to develop novel computational models (kinetic, constraint-based, and machine learning models) to infer metabolic activities within the human gut microbiome. Our focus extends beyond predicting overall fecal metabolomics to understanding the metabolic behavior of individual microbes. **We ask: given microbiota multi-omics data, can we infer which microbes are producing or uptaking which metabolites, and how do these activities change over time?** By predicting these metabolic activities, we can map out microbial interactions (e.g., nutrient competition, metabolic cross-feeding) difficult to measure experimentally. This approach offers a new perspective on understanding microbial dysbiosis by uncovering altered metabolic activities (rather than just metabolite levels) in patients, such as those with cystic fibrosis.

## Direction 2: Computational fungal metabolomics and fluxomics
<img src="../images/Theme2_schematic.png" align="right" width="500px"/>
Fungal pathogens adapt and survive through dynamic changes in their metabolism, which play a key role in their ability to cause disease. While recent omics technologies such as transcriptomics have revolutionized our ability to characterize cellular states, measuring metabolism remains challenging because it involves complex and changing reaction rates. Accurate quantification of these metabolic fluxes at the genome scale could transform how we study infections and identify new targets for antimicrobial therapies. **Our project aims to develop the first fully automated, genome-scale platform that processes raw gas/liquid chromatography-mass spectrometry (GC/LC-MS) data from isotope tracing experiments to provide genome-wide metabolic flux measurements**. This breakthrough will advance infectious disease research and systems biology by enabling routine analysis of metabolic dynamics in microbes and hosts. Using this technology, we aim to address clinically important questions related to fungal gut colonization, drug resistance, and infections. For example, how do *Candida albicans* rewire its metabolism during its pathogenic yeast-to-hyphae transition?

## Direction 3: Colonization resistance and gut-borne infections
<img src="../images/Theme3_schematic.png" align="right" width="500px"/>
The intestinal microbiota is a major source of invasive microbial infections in immunocompromised patients. Despite routine administration of prophylactic antimicrobial drugs to prevent these infections, breakthrough infections caused by drug-resistant pathogens in the intestine remain a significant and life-threatening complication. One critical risk factor of these gut-borne infections is the loss of protective gut commensal bacteria, which provide colonization resistance against harmful pathogens. We have previously showed that a higher abundance of <i>Klebsiella oxytoca</i> in the gut is associated with a lower risk of <i>Escherichia coli</i> infections among patients receiving allogeneic hematopoietic cell transplantation. We further demonstrated that clinical isolates of <i>K. oxytoca</i> outcompete <i>E. coli</i> in the mouse gut. We will use the two-species community as the model system to study the dynamic role of microbial ecology in gut colonization resistance and infections. We will focus on ecological mechanisms such as nutrient competition and toxin production as well as the regulatory effects of the gut microbiota on these mechanisms.


