---
permalink: /research/
title: "Research Directions"
author_profile: true
redirect_from: 
  - /md/
  - /markdown.html
---

## Direction 1: Metabolic modeling of human gut microbiome
<img src="../images/Theme1_schematic.png" align="left" width="300px"/>
We aim to develop novel computational models (kinetic, constraint-based, and machine learning models) to infer metabolic activities within the human gut microbiome. Our focus extends beyond predicting overall fecal metabolomics to understanding the metabolic behavior of individual microbes. <strong>We ask: given microbiota multi-omics data, can we infer which microbes are producing or uptaking which metabolites, and how do these activities change over time?</strong> By predicting these metabolic activities, we can map out microbial interactions (e.g., nutrient competition, metabolic cross-feeding) difficult to measure experimentally. This approach offers a new perspective on understanding microbial dysbiosis by uncovering altered metabolic activities (rather than just metabolite levels) in patients, such as those with cystic fibrosis.

## Direction 2: Computational fungal metabolomics and fluxomics
<img src="../images/Theme2_schematic.png" align="left" width="500px"/>
Fungal pathogens adapt and survive through dynamic changes in their metabolism, which play a key role in their ability to cause disease. While recent omics technologies such as transcriptomics have revolutionized our ability to characterize cellular states, measuring metabolism remains challenging because it involves complex and changing reaction rates. Accurate quantification of these metabolic fluxes at the genome scale could transform how we study infections and identify new targets for antimicrobial therapies. Our project aims to develop the first fully automated, genome-scale platform that processes raw gas/liquid chromatography-mass spectrometry (GC/LC-MS) data from isotope tracing experiments to provide genome-wide metabolic flux measurements. This breakthrough will advance infectious disease research and systems biology by enabling routine analysis of metabolic dynamics in microbes and hosts. Using this technology, we aim to address clinically important questions related to fungal gut colonization, drug resistance, and infections, including:  
1. Why do some fungal strains cause infections in patients, while others remain harmless in the environment?
2. The yeast-to-hyphae transition is essential for <i>C. albicans</i> to invade host tissues. How does metabolism differ between these two morphological forms?
3. Fungi that develop antifungal resistance often acquire mutations in drug targets, which can reduce their fitness. How do they rewire their metabolism to compensate for this cost?
4. Beyond direct drug targets, cellular metabolism can additionally regulate antifungal drug efficacy. Which metabolites and metabolic pathways contribute to drug resistance?**

## Theme 3: Colonization resistance and gut-borne infections
<img src="../images/Theme3_schematic.png" align="left" width="500px"/>
<p>The intestinal microbiota is a major source of invasive microbial infections in immunocompromised patients, particularly those undergoing allogeneic hematopoietic cell transplantation. Despite routine administration of prophylactic antimicrobial drugs to prevent these infections, breakthrough infections caused by drug-resistant pathogens in the intestine remain a significant and life-threatening complication. One critical risk factor of these gut-borne infections is the loss of protective gut commensal bacteria, which provide colonization resistance against harmful pathogens.</p>
	
[Our previous work](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(23)00220-2?dgcid=raven_jbs_etoc_email) showed that a higher abundance of <i>Klebsiella oxytoca</i> in the gut is associated with a lower risk of <i>Escherichia coli</i> infections. Our preliminary data demonstrated that clinical isolates of <i>K. oxytoca</i> outcompete <i>E. coli</i> in the mouse gut. We will use the two-species community as an example to study the role of microbial ecology in gut colonization resistance and infections. The observed colonization resistance can be attributed to nutrient competition and the production of DNA-alkylating enterotoxin tilimycin by <i>K. oxytoca</i>. Additionally, other gut commensal bacteria may influence <i>E. coli</i> colonization by interacting with both species, possibly by altering nutrient competition and tilimycin production. We will combine Biolog phenotyping, multi-omics profiling, and metabolic modeling to investigate the following questions:

**1. What nutrients favors the growth of <i>K. oxytoca</i> over <i>E. coli</i> in aerobic and anaerobic conditions?**

**2. What nutrients enable tilimycin production by <i>K. oxytoca</i>, and what is the associated signaling pathway?**

**3. What gut commensals are inhibited by tilimycin, and is the growth inhibition species- or strain-specific?**

<p>Despite the simplicity of the model system, the interactions between the two bacteria are expected to be both dynamic and complex. Understanding these interactions is essential for identifying potential therapeutic strategies to restore protective gut microbiota in immunocompromised patients, where traditional antibiotics often fail. Moving forward, we plan to increase the complexity of the model system by incorporating additional gut commensals.</p>


